ppr-ui/src/App.vue

```diff
@@ -235,7 +235,7 @@ export default defineComponent({
         // When we are authenticated, allow time for session storage propagation from auth, then initialize application
         // (since we won't get the event from Signin component)
         if (isAuthenticated.value) {
-          onProfileReady(true)
+          setTimeout(() => { onProfileReady(true) }, 3000)
```

Comment:
3s? Would it slow down the loading of the page or in the background?

Reply:
It would be the overlay state, the app loads up but the contents (ie components in the router-view) are on a delay. 

The jist: SbcHeader, the auth common component kicks off some auth related fetching/storing etc when it's mounted. We are giving it some time before rendering everything inside the router-view and proceeding with some other functionality that relies on that info from Auth being in the store.

Reply:
I'll probably reduce to 1s, just wanted to confirm it was the issue first.